courses/free-courses-ml.md

```diff
@@ -68,6 +69,12 @@
 * [Compiler Design](https://www.youtube.com/playlist?list=PLwe8bwPXjlLtPlbzxU_SICYI3PeYpEAtJ) -  Learn CSE Malayalam
 
 
+### Cybersecurity
+
+* [Cyber Security Tutorial Malayalam](https://www.youtube.com/playlist?list=PLR2UNjW_Pkm8LUpryeuiLMpEf4KgCDeBu) -  RootSaid - Robotics, Technology & Cyber Security
```

Comment:
```suggestion
* [[Cyber Security Tutorial Malayalam \| Ethical Hacking Courses for IT Career](https://www.youtube.com/playlist?list=PLR2UNjW_Pkm8LUpryeuiLMpEf4KgCDeBu) -  RootSaid
```
Don't invent titles! Use the title given by the resource.

Reply:
Resolved.